DOCKER.md

```diff
@@ -142,27 +163,70 @@ Or if you prefer using docker directly:
         "SBCTL_TOKEN=${SBCTL_TOKEN}",
         "-e",
         "MCP_BUNDLE_STORAGE=/data/bundles",
-        "mcp-server-troubleshoot:latest",
-        "python",
-        "-m",
-        "mcp_server_troubleshoot.cli"
-      ],
-      "env": {}
+        "-e",
+        "MCP_KEEP_ALIVE=true",
+        "mcp-server-troubleshoot:latest"
+      ]
     }
   }
 }
 ```
 
 Replace `${LOCAL_BUNDLE_DIRECTORY}` with the actual path to your bundles directory. Make sure the `SBCTL_TOKEN` environment variable is set in your environment if needed.
 
+### Enhanced Configuration Options
+
+#### Custom Bundle Directory
+
+To specify a custom bundle directory without listing all the Docker arguments:
+
+```json
+{
+  "mcpServers": {
+    "troubleshoot": {
+      "command": "docker",
+      "args": [
+        "run",
+        "-i",
+        "mcp-server-troubleshoot:latest"
+      ],
+      "bundleDir": "/path/to/your/bundles"
+    }
+  }
+}
+```
+
+#### Environment Variables
+
+To pass environment variables to the server:
```

Comment:
We shouldn't recommend putting a token in your config file. This section should be removed, the user can use the "-e" example above to pass in tokens.